src/styles/index.css

```diff
@@ -85,20 +85,21 @@ html .akEditor > div:first-child > div:first-child {
     overflow: visible; /* important for dropdowns */
 }
 
-/* avatars */
+/* second header block (contains sharing banner) */
```

Comment:
This styling could be extracted in a proper stylus file but I thought it was more idiomatic to follow the initial convention of hacking editor-related things in this file. Thoughts?